### PR TITLE
Fix bug for lstmp

### PR DESCRIPTION
**Pull request by xuezhong**
1 add cell_clip
2 add proj_clip
3 fix bug of h0
4 change default proj_activation to identity
test=develop

### CODE REVIEW

**Comment by kuke** [comment]
Please add `test=develop` in commit message to trig ci

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -770,6 +780,11 @@ def dynamic_lstmp(input,
                               the bias is initialized zero. Default: None.
         use_peepholes(bool): Whether to enable diagonal/peephole connections,
                              default `True`.
+        cell_clip(float): If provided the cell state is clipped
```
Comment (kuke) [inline python/paddle/fluid/layers/nn.py]: How about another name like `cell_clip_threshold`, and adding an assert `assert(cell_clip_threshold>=0)` ?

File: paddle/fluid/operators/lstmp_op.h [hunk 2]
```diff
@@ -60,16 +94,37 @@ class LSTMPKernel : public framework::OpKernel<T> {
       PADDLE_THROW("unsupported activation type");
   }
 
+  void Print(const Tensor& t, std::string name) const {
```
Comment (kuke) [inline paddle/fluid/operators/lstmp_op.h]: Can remove this function？

Comment (xuezhong) [reply paddle/fluid/operators/lstmp_op.h]: yes just a function for debug

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (xuezhong) [reply python/paddle/fluid/layers/nn.py]: this parameter uses the same name in API of tensorflow rnn cell, so the name may be more convenient to user.

Comment (xuezhong) [reply python/paddle/fluid/layers/nn.py]: can add assert(cell_clip >=0)

**Comment by xuezhong** [comment]
> Please add `test=develop` in commit message to trig ci

ok

**Review by kuke — DISMISSED** [review]
LGTM

**Review by panyx0718 — APPROVED** [review]

**Review by kuke — APPROVED** [review]

**Pull request merged**